packages/camera/camera_windows/windows/record_handler.h

```diff
@@ -16,6 +16,8 @@ namespace camera_windows {
 using Microsoft::WRL::ComPtr;
 
 enum class RecordingType {
+  // Camera is not recording.
+  kNone,
```

Comment:
This new state was added so that `IsContinuousRecording` and `IsTimedRecording` returns false if the recording isn't running or stopping. Technically this could've been achieved by checking `recording_state_`, but, I'd rather have `type_` be a sensible value if the recording is stopped.